app/devine/chargepoint/config.py

```diff
@@ -0,0 +1,28 @@
+# Site = [SLAC, Google]
+# Groups = [slac_GISMO, slac_B53, google_plymouth, google_B46, google_CRIT]
+
+# slac_api -> slac_B53
+# slac-gismo_api -> slac_GISMO
+
+chargepoint_api_keys = {
+    'slac_api_username':'92d0023b4f07baa1857381a51de78772546674ef2b8391416000751',
+    'slac_api_password':'c2f058fc77298198f5bb3089cb354053',
+    'google_api_username': 'a0a3b3a5db088eef682a0252745dc5cf5f776a22a3ac71601661474',
+    'google_api_password':'7c0d2a84d442598fe5ea2db2cab71e97',
+    'slac-gismo_api_username': '7f74b2f7226e5cf50d59e84cc628649f5dadd5a5234f51571673509',
+    'slac-gismo_api_password': 'eb07d5826fb96b82efb9c130d26cafa6'
```

Comment:
We need to remove passwords from github. We could pull this from env vars

Reply:
Ok, I see. Could you please tell me how to pull the password from env vars? I'm not familiar with that. Thank you!